dvc/command/get.py

```diff
@@ -44,10 +44,10 @@ def add_parser(subparsers, parent_parser):
     get_parser.add_argument(
         "url", help="URL of Git repository with DVC project to download from."
     )
-    get_parser.add_argument("path", help="Path to data within DVC repository.")
     get_parser.add_argument(
-        "-o", "--out", nargs="?", help="Destination path to put data to."
+        "path", help="Path to a file or directory within a DVC repository."
```

Comment:
Same Q as https://github.com/iterative/dvc/pull/2837#discussion_r351034835: Do we support Git-tracked directories?

Reply:
Yes

Reply:
SU2?